packages/components/src/utils/jwtToken.ts

```diff
@@ -0,0 +1,9 @@
+export default function getMaxAge(token: string): number {
+  try {
+    const content = JSON.parse(atob(token.split('.')[1]));
+    // value in seconds
+    return +content.exp - +content.iat;
```

Comment:
What is the use of unary plus operators here? Is it used for number conversion? Maybe something else would be clearer if you want to ensure that the `content.exp` and `content.iat` are converted to numbers. Or is this a normal way to do this in Javascript?

Reply:
i think its a normal way. used it many times

Reply:
As it is the code is not self-evident or apparent, for a comparison the `!!x` idiom for boolean conversion is recommended by the airbnb javascript style guide, but I didn't find any evidence for the use of unary plus operator for string to number conversion being recommended by any prominent javascript style guides (used copilot for checking those so it's possible it missed something though). But anyway, it'd be good to add clarity to this code IMHO.

Reply:
fixed